<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiberaSparkDataWindow</class>
 <widget class="QMainWindow" name="LiberaSparkDataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libera Spark</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TBT_IQ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QEPlotter" name="plot">
      <property name="toolBarIsVisible" stdset="0">
       <bool>false</bool>
      </property>
      <property name="pvItemsIsVisible" stdset="0">
       <bool>true</bool>
      </property>
      <property name="statusIsVisible" stdset="0">
       <bool>false</bool>
      </property>
      <property name="DataVariableA" stdset="0">
       <string>SR-DI-LBS:ADC:A</string>
      </property>
      <property name="SizeVariableX" stdset="0">
       <string>SR-DI-LBS:ADC:Size</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Buffer Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QENumericEdit" name="txtBuffer">
        <property name="autoScale">
         <bool>true</bool>
        </property>
        <property name="precision">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>678</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="titleGroupBox_2">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>792</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPlotter</class>
   <extends>QEFrame</extends>
   <header>QEPlotter.h</header>
  </customwidget>
  <customwidget>
   <class>QEFrame</class>
   <extends>QFrame</extends>
   <header>QEFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
